.eslintignore

```diff
@@ -21,9 +21,4 @@ experimental/babel-preset-env/test/debug-fixtures
 packages/babel-standalone/babel.js
 packages/babel-standalone/babel.min.js
 packages/babylon/build
-packages/babylon/scripts
 packages/babylon/test/expressions
-
-# Prettier tries to insert trailing commas in function calls, which Node.js
-# doesn't natively support. This causes an error when loading the Gulp tasks.
-packages/babel-standalone/src/gulpTasks.js
```

Comment:
Does not exist anymore?

Reply:
I think it was merged into the main Gulpfile.